pkg/actor/initialize.go

```diff
@@ -127,9 +128,32 @@ func (init initialize) Act(ctx context.Context, cluster *resource.Cluster) error
 		return errors.Wrap(err, msg)
 	}
 
-	cluster.SetTrue(api.InitializedCondition)
+	fetcher := resource.NewKubeFetcher(ctx, cluster.Namespace(), init.client)
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
How are we getting conflicts?  I don't disagree that we should retry, but how would we need to?

Reply:
Added a comment giving an example of how this might happen